pom.xml

```diff
@@ -56,4 +78,16 @@
             <email>janarioliver@gmail.com</email>
         </developer>
     </developers>
+
+    <build>
+        <pluginManagement>
```

Comment:
I removed the `pluginManagement` element and it also works, so I am removing it. Thanks.